Sources/ATTNAPI.m

```diff
@@ -42,6 +95,95 @@ - (void)sendUserIdentity:(ATTNUserIdentity *)userIdentity domain:(NSString *)dom
     }];
 }
 
+- (void)sendEvent:(id<ATTNEvent>)event userIdentity:(ATTNUserIdentity*)userIdentity domain:(NSString*) domain {
+    [self getGeoAdjustedDomain:domain completionHandler:^(NSString* geoAdjustedDomain, NSError* error) {
```

Comment:
I said it above but repeating here just to make sure. I think if possible we want to hold domain in memory to not need it a second time

Reply:
Yeah, domain makes sense to me. That shouldn't change after startup. I'll add to the constructor.

Reply:
And in the future we'll store the geoAdjustedDomain as a member variable as well